diagrams/cs-for-hackers.js

```diff
@@ -0,0 +1,117 @@
+var nameSpace = 'template';
+
+var svgWidth  = 250;
+var svgHeight = 250;
+
+var xOffset = 40;
+
+
+// dynamic values
+
+var textColor = '#555555';
+
+//table appearance
+var rowHeight = 30;
+var rowWidth = 110;
+var numRows = 6;
+// var rowColor = '#eff9fc';
+var strokeColor = '#555555';
```

Comment:
same thing. you don't need this. every stroke will be this color by default. you don't even have to pass in an argument